src/pages/grants/grantsLanding/GrantsLandingPage.tsx

```diff
@@ -22,76 +22,69 @@ export const GrantsLandingPage = () => {
   }
 
   return (
-    <VStack paddingTop={{ base: '10px', lg: '20px' }} bg="neutral.0" minHeight="100%" width="100%" alignItems="center">
-      <VStack
-        my={{ base: 0, lg: 5 }}
-        px={{ base: '10px', lg: '0px' }}
-        spacing="20px"
-        width={isMobile ? '100%' : '909px'}
-      >
-        <VStack spacing="10px">
-          <Text fontSize={isMobile ? '4xl' : '44px'} fontWeight="medium" textAlign="center">
-            🥳
-          </Text>
-          <Text
-            fontSize={'25px'}
-            fontWeight="bold"
-            textAlign="center"
-            fontFamily={fonts.header}
-            textShadow={' 0px 0px 25.7663px rgba(22, 232, 194, 0.11)'}
-            color={'primary.500'}
-            textTransform="uppercase"
-          >
-            {t('Geyser Grants')}
-          </Text>
-          <H2 textAlign="center" paddingY="5px" fontSize="44px" fontWeight="700">
-            {t('Empowering bitcoin creators')}
-          </H2>
-          <H3 textAlign="center" color={'neutral.600'}>
-            {t('Funding educators, creatives and builders doing Bitcoin-only projects on Geyser.')} <br />{' '}
-            {t('Funded by bitcoiners who want to change the world.')}
-          </H3>
-        </VStack>
+    <>
+      <VStack spacing="10px">
+        <Text fontSize={isMobile ? '4xl' : '44px'} fontWeight="medium" textAlign="center">
+          🥳
+        </Text>
+        <Text
+          fontSize={'25px'}
+          fontWeight="bold"
+          textAlign="center"
+          fontFamily={fonts.header}
+          textShadow={' 0px 0px 25.7663px rgba(22, 232, 194, 0.11)'}
+          color={'primary.500'}
+          textTransform="uppercase"
+        >
+          {t('Geyser Grants')}
+        </Text>
+        <H2 textAlign="center" paddingY="5px" fontSize="44px" fontWeight="700">
+          {t('Empowering bitcoin creators')}
+        </H2>
+        <H3 textAlign="center" color={'neutral.600'}>
+          {t('Funding educators, creatives and builders doing Bitcoin-only projects on Geyser.')} <br />{' '}
+          {t('Funded by bitcoiners who want to change the world.')}
+        </H3>
```

Comment:
use new Headings, from shared/components/typography

Reply:
This is deprecated.